go-controller/pkg/util/util_unit_test.go

```diff
@@ -0,0 +1,242 @@
+package util
+
+import (
+	"bytes"
+	"fmt"
+	mock_k8s_io_utils_exec "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/testing/mocks/k8s.io/utils/exec"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util/mocks"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"net"
+	"testing"
+)
```

Comment:
L4-11: can you please group the imports like we have done everywhere else in the project? standard library followed by github.com

Reply:
good catch, shall fix it.